CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## \[Unreleased\]
 
 ### Added
+- More intro text
```

Comment:
I don't think this needs to be in the changelog and might even confuse people.

Reply:
Fair. I was going back and forth on it. Wouldn't have added it except I already had a line there. Will take it out.